web/client/components/share/SharePanel.jsx

```diff
@@ -198,6 +223,67 @@ class SharePanel extends React.Component {
                         })}>
                     <Message msgId="share.showHomeButton" />
                 </Checkbox>}
+                {
+                    this.props.isScrollPosition
+                    && this.props.advancedSettings.sectionId
+                    && <Checkbox
+                        checked={this.props.settings.showSectionId}
+                        onChange={() =>
+                            this.props.onUpdateSettings({
+                                ...this.props.settings,
+                                showSectionId: !this.props.settings.showSectionId
+                            })}>
+                        <Message msgId="share.showSectionId" />
+                    </Checkbox>
+                }
+                {this.props.settings.centerAndZoomEnabled && <div>
```

Comment:
I compared this file diff with the one on merged on master https://github.com/geosolutions-it/MapStore2/pull/5637/files and it seems there is code that it's not part of the issue. We should cherry pick only code related to the add story section URL fix